src/pyedb/dotnet/edb_core/components.py

```diff
@@ -785,6 +785,9 @@ def create_port_on_pins(self, refdes, pins, reference_pins, impedance=50.0, port
             reference_pins = [reference_pins]
         if isinstance(refdes, str) or isinstance(refdes, EDBComponent):
             refdes = self.instances[refdes]
+        if refdes.rlc_values:
```

Comment:
It is better to check type rathen than rlc value. Why is RLC not supported?

Reply:
Yes but type is EDBComponent for all inputs (even inductors, caps etc). Ports are not successfully created on RLC pins. Should this be added for 3DL?

Reply:
@gkorompi there is a command to replace RLC by ports which deactivate component and create a circuit port. You might want to call this one when you hit such component ? 
Also also rather component type instead of rlc_value.